### ISSUE TITLE
Convention: PHP scripts as a standalone file

### ISSUE THREAD

**Issue by DrAlbertCruz** [namesEd/Team-Ramrod#13]
This is related to the previous post about keeping CSS, JS and HTML separate.

If you have PHP code that is essentially running a script and doesn't affect the user's front-end viewing experience you should put it into a separate PHP file. 

And example is @namesEd `login.php`. Having this as a PHP file is fine, but notice how lines 8 - 30 are essentially a standalone PHP script?

The convention here is to use Jquery/AJAX to POST the form data to a standalone PHP file that contains just lines 8-30 (let us call it `login_script.php`, and to have this new file return a value to indicate success or failure. Your `login.php` would have hidden CSS elements for alert boxes that may say "login success!" or "failure" based on the return code from `login_script.php`.

TLDR: Pure PHP scripting should be in its own PHP file, and you will need to use Jquery/AJAX to post into this script, and then add some elements like alert boxes to let the user know that its working. 

By the way doing this allows you to separate front-end devs and back-end devs workflows. You establish an extraction where one person focuses on the Jquery front end and the other such as @namesEd just focuses on the functionality of the `login_script.php`.

- [How to POST something to a PHP using AJAX/jquery](https://stackoverflow.com/questions/5004233/jquery-ajax-post-example-with-php)
- [What is JSON, the intermediate file format used when POST-ing something using this AJAX/Jquery/PHP workflow](https://www.w3schools.com/whatis/whatis_json.asp)
- [How to return data from a pure PHP script through AJAX/jquery](https://stackoverflow.com/questions/2410773/how-to-return-data-from-php-to-a-jquery-ajax-call)
